src/test/packages/00-archive-path/zarf.yaml

```diff
@@ -0,0 +1,59 @@
+# yaml-language-server: $schema=../../../../zarf.schema.json
+kind: ZarfPackageConfig
+metadata:
+  name: archive-path
+
+x-untar: &base-untar
+  required: true
+  files:
+    - source: archive.tar
+      target: src/test/packages/00-archive-path/output.txt
+      shasum: 03ba204e50d126e4674c005e04d82e84c21366780af1f43bd54a37816b6ab340
+      extractPath: archive/content/data.txt
+  actions:
+    onCreate:
+      before:
+        - cmd: mkdir -p archive/content
+          mute: true
+        - cmd: echo "Hello World!" > archive/content/data.txt
+          mute: true
+        - cmd: tar -cf archive.tar archive
+          mute: true
+      onFailure:
+        - cmd: rm -f archive/content/* archive.tar
+      after:
+        - cmd: rm -f archive/content/* archive.tar && rmdir archive/content && rmdir archive
+          mute: true
+
+components:
+  - name: untar
+    only:
+      flavor: linux
```

Comment:
We should use `only.localOS` here

Reply:
I had it there before but for some reason it wasn't applying on my windows machine, ill try again